magma_access_gateway_installer/agw_network_configurator.py

```diff
@@ -112,7 +113,7 @@ def _dns_configured(self) -> bool:
         """Checks whether DNS has already been configured."""
         with open(self.ETC_SYSTEMD_RESOLVED_CONF_PATH, "r") as dns_config:
             for line in dns_config.readlines():
-                if f"DNS={self.DNS_ADDRESSES}" in line:
+                if f"DNS={self.dns_ips_list}" in line:
```

Comment:
Would this return true if the list was in a different order?

Reply:
Of course not. That's stupid.
Refactored.